src/platform/utilities/sso/index.js

```diff
@@ -54,15 +37,11 @@ export async function ssoKeepAliveSession() {
 }
 
 export async function checkAutoSession() {
-  const [{ ttl, authn }, userProfile] = await Promise.all([
-    ssoKeepAliveSession(),
-    vaGovProfile(),
-  ]);
-  if (userProfile?.signIn?.ssoe) {
+  const { ttl, authn } = await ssoKeepAliveSession();
+  if (hasSession()) {
```

Comment:
`s/hasSession/hasSessionSSO/` ?

Reply:
Neither `hasSession` nor `hasSessionSSO` seems to be equivalent to the condition that this is replacing. 

`hasSessionSSO` indicates that an SSOe session is present based on the keepalive call that just completed.
`hasSession` indicates that a VA.gov session is present but not necessarily that the user was signed in with SSOe.

Reply:
i.e. we don't have access to the `signIn.ssoe` state here without the vaGovProfile call. 
The `setupProfileSession` in `platform/user/profile/utilities/index.js` does have access to the userProfile (it's setting the signIn type in Sentry), but I don't know if we want to put in yet another local storage item, or somehow plumb the redux state into this code.

Reply:
Or pass it in from `AutoSSO`...but I don't know if I'm recycling what you used to have for these components.

Reply:
That's right, this forgets that edge case. Before going the user endpoint route, I was deriving the logic from the `ssoe` prop on the original `/user` call. I think we can still use that and just account for an additional comparison in the `AutoSSO`'s `componentDidUpdate` that lets us call keepAlive after the user call is initialized